cmd/fissh/main.go

```diff
@@ -86,8 +98,13 @@ func teaHandler(s ssh.Session) (tea.Model, []tea.ProgramOption) {
 	appStyle := renderer.NewStyle().AlignHorizontal(lipgloss.Center)
 
 	ip := s.RemoteAddr().(*net.TCPAddr).IP
-	timezone := timezone.LookupTimezone(ip.String())
-	loc, err := time.LoadLocation(timezone)
+
+	foundTimeZone, err := extractTimezoneFromEnv(s.Environ())
+	if err != nil {
```

Comment:
this just protects us if the timezone wasn't specified, would we be able to make this the case if the timezone in the environment isnt valid?

Reply:
Sure! Is this because not finding the TIMEZONE env value isn't actually an error case? And I've seen it a few other places but is returning an empty string if the env var isn't found the correct alternative in this case?